specs/eventing/overview.md

```diff
@@ -1,97 +1,100 @@
 
-- A **Trigger** describes a filter on event attributes which should be delivered
-  to an _Addressable_.
+- Content-based event routing ([`eventing.knative.dev`](#eventing))
 
-- A **Broker** provides a bucket of events which can be selected by attribute.
+Knative Eventing does not directly specify mechanisms for other event-processing
+models, including multi-stage workflows, correlated request-reply, and
+sequential (windowed) event processing; these models could be built using the
+primitives provided by Knative, or Knative could deliver events to an external
+system that implements these models.
 
-<!-- https://drive.google.com/open?id=1CXRvT2g6sxk6-ZrwYcSf2BahCNVlLTLNZkm-laQitMg -->
+In addition to the primitives needed to express the above patterns, Knative
+Eventing defines two [_interface contracts_](#interface-contracts) to allow
+connecting multiple types of Kubernetes objects as event senders and recipients
+to the core primitives.
 
-![Broker Trigger Overview](images/broker-trigger-overview.svg)
+<!-- TODO: add a drawing -->
 
-The above diagram shows a _Broker_ ingesting events and delivering to a
-_Service_ only when the _Trigger_ filter matches.
+## Interface Contracts
 
-- A **Subscription** describes the transformation of an event and optional
-  forwarding of a returned event.
+In addition to the concrete types described below in the `messaging.knative.dev`
+and `eventing.knative.dev` API groups, Knative Eventing supports referencing
+objects in other API groups as destinations for event delivery. This is done by
+defining partial schemas which the other resources must support. The following
+interface contracts define object fragments and partial schemas (required fields
+on an arbitrary API object) which form a basis for Knative Eventing.
 
-- A **Channel** provides event persistence and fanout of events from a
-  well-known input address to multiple outputs described by _Subscriptions_.
+### Addressable
 
-<!-- This image is sourced from https://drive.google.com/open?id=10mmXzDb8S_4_ZG_hcBr7s4HPISyBqcqeJLTXLwkilRc -->
+**Addressable** resources expose a resource address (HTTP URL) in their `status`
+object. The URL is used as a destination for delivery of events to the resource;
+the exposed URL must implement the [data plane contract](data-plane.md) for
+receiving events.
 
-![Resource Types Overview](images/resource-types-overview.svg)
+[**Broker**](#broker) and [**Channel**](#channel) both implement **Addressable**
+to receive events from other components.
 
-Channels as well as Sources are defined by independent CRDs that can be
-installed into a cluster. Both Sources and Channels implementations can be
-created directly. A Channel however offers also a way to create the backing
-implementation as part of the generic Channel (using a _channelTemplate_).
+### Destination
 
-## Trigger
+**Destination** is an interface (object fragment) which is used consistently
```

Comment:
The term _interface_ IMO is too "language specific", I prefer contract when referring to `Addressable` and `Destination`

Reply:
Makes sense